test/integration/backward_compatible/parallel/topic/ReliableTopicTest.js

```diff
@@ -189,4 +189,38 @@ describe('ReliableTopicTest', function () {
         const obj = clientOne.getSerializationService().toObject(item.payload);
         expect(obj).to.equal(11);
     });
+
+    it('writes and reads messages using async methods', async function () {
+        const topic = await clientOne.getReliableTopic('async');
+
+        await topic.addListener(() => {});
```

Comment:
why add a listener here that is not used?

Reply:
There was something in my head but not implemented and missed. Done.